pkg/vsphere/tasks/waiter.go

```diff
@@ -26,6 +29,10 @@ import (
 	"github.com/vmware/vic/pkg/errors"
 )
 
+const (
+	maxBackoffFactor = int64(8)
```

Comment:
If I read this correctly we max out at a retry of 1sec (150ms * 8)  - I'd let this grow to at least 3sec given potential completion times in busy environments.